### PR TITLE
Add clarification about how composite service discovery works.

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: docs/docs/service-discovery/composite.md [hunk 1]
```diff
@@ -38,9 +38,14 @@ For each service that should consist of multiple services, configure the service
     quarkus.stork.serviceB.service-discovery.type=...
     ```
 
-
 Remember to define the services that make up your composite service.
 
+Be aware that Stork doesn't work as a standalone service discovery cluster. 
+Instead, it processes composite configurations—meaning it handles multiple configurations that may use different service discovery implementation. 
+For each specific configuration, Stork delegates the service discovery task to the appropriate service discovery provider. 
+If one of these providers doesn't respond or fails, it affects Stork's ability to resolve that specific configuration, but it doesn't mean Stork itself is faulty;
```
Comment (cescoffier) [inline docs/docs/service-discovery/composite.md:46]: That's the current implementation, but shouldn't we only fail if none of the sub-discovery found a matching instance or if all fail?

Basically, if one fails but if another one provides a service instance, it should pass this service instance.

**Review by cescoffier — APPROVED** [review]
LGTM, added a comment because I believe we can improve the composite service discovery.

File: docs/docs/service-discovery/composite.md (hunk 1 above)
Comment (aureamunoz) [reply docs/docs/service-discovery/composite.md:46]: Yes, we can better manage the error case.
Do we do it in this PR? or shall we create an issue and do it in another one?

**Comment by cescoffier** [comment]
In another one.

**Pull request merged**